docs/containers/03.handsondocker.rst

```diff
@@ -151,34 +151,34 @@ Alternatively, you could also do this *interactively*
 
 **Install your tool in the image**
 
-.. code-block:: bash
+.. code-block:: console
 
    root@ded8d40f1a1e:/#
-   # install dependencies
+   install dependencies
```

Comment:
does this look better on readthedocs without the '#'?

Reply:
Yes, I was testing this locally with the RTD theme as I was updating. With the 'console' code-block, anything that's not a prompt gets greyed out which looks nice. Unfortunately, the '#' is a typical root prompt, so it gets colored like a prompt.